openr/fib/Fib.cpp

```diff
@@ -271,6 +277,14 @@ Fib::getUnicastRoutesFiltered(std::vector<std::string> prefixes) {
   return retRouteVec;
 }
 
+void
+Fib::compareDefaultRoutes(const thrift::UnicastRoute& oldRoute, const thrift::UnicastRoute& newRoute) {
+  if (oldRoute.dest == kDefaultRoutePrefix && oldRoute.nextHops != newRoute.nextHops) {
+    stats.countDefaultRouteChanged_++;
+    VLOG(2) << "Default route changed. Number of new next hops: " << newRoute.nextHops.size();
+  }
+}
```

Comment:
Are those changes specifically for TG? I am NOT aware of anything like this from openr code base.

Reply:
Yeah, this is a default route watching stats TG wanted ... If it's not evasive/costly we could look to merge.